docs/index.rst

```diff
@@ -71,11 +86,17 @@ similarly, the user will be warned. These warnings are not necessarily a reason
 for concern, they are there to help identifying issues. In these cases, we
 recommend that you check if the files in question were indeed mapped to the
 expected place, and open a bug report if they weren't.
-If the project includes shared libraries that are needed by other files, those
-libraries will be included in the wheel, and native files will have their
+
+If the project itself includes shared libraries that are needed by other files,
+those libraries will be included in the wheel, and native files will have their
 library search path extended to include the directory where the libraries are
 placed.
 
+If the project depends on external shared libraries, those libraries will not
+be included in the wheel. Vendoring those libraries or finding a suitable runtime
+dependency that provides those libraries is the responsibility of the user
+of ``meson-python``.
```

Comment:
The user of `meson-python` would be the package developer, not necessarily the people trying to compile or install the project. As such, this sentence seems a bit confusing, because the people that need to find the runtime dependency are the ones compiling/installing the project.

Here's my suggestion.
```suggestion
be included in the wheel. Vendoring those libraries or finding a suitable runtime
dependency that provides those libraries is the responsibility of the user
of binary distributions built by ``meson-python``.
```

Maybe we can also point people that want to distribute their artifacts to `auditwheel` and `delocate`?

Reply:
Ah, good point. With "user" I actually meant the package developer. But it could be either, so this needs a bigger rephrase.
- The package developer should vendor those libraries or find a suitable runtime dependency, OR
- The user must install that shared library in some other way (e.g., through a distro package manager).

Reply:
For SciPy, we want to do the vendoring, and how to tweak the vendoring mechanism is a little TBD right now.

Reply:
Okay, I went for an extended version of the above and also included auditwheel/delocate.